modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -327,9 +335,21 @@ definition {
 
 			CommerceEntry.clickEntryDropdownMenuItem(
 				entryName = "Value1",
-				menuItem = "Toggle Default");
+				menuItem = "Edit");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			Uncheck.uncheckToggleSwitch(
+				checkboxName = "Default",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			Button.clickSave();
 
 			Alert.viewSuccessMessage();
+
+			SelectFrameTop();
+
+			Click(locator1 = "Icon#CLOSE");
```

Comment:
After saving, please add an assert to verify if the toggle has successfully changed the status.